lib/collect.js

```diff
@@ -0,0 +1,18 @@
+import { getUserContext } from "./init";
+
+
+export async function collectStory(storyId) {
+    const userContext = await getUserContext();
+    const [profileId, pubId] = storyId;
+
+    userContext.lensContract.methods
+        .collect(
+            profileId,
+            pubId,
+            []
+        )
+        .send({ from: userContext.userAccount, gasPrice: process.env.NEXT_PUBLIC_POST_GAS_PRICE })
+        .on('transactionHash', console.log)
+        .on('receipt', console.log)
+        .on('error', console.error);
```

Comment:
Would be nice to show some kind of feedback to the user after a successful collection. For example, using https://yarnpkg.com/package/react-toastify, or changing button text to something like "Done". Can be added separately

Reply:
Good idea. I'll try to add this separately later